crates/workspace/src/pane.rs

```diff
@@ -1119,6 +1121,28 @@ impl Pane {
         )
     }
 
+    fn copy_path(&mut self, _: &CopyPath, cx: &mut ViewContext<Self>) {
+        if let Some(clipboard_text) = self
+            .active_item()
+            .as_ref()
+            .and_then(|entry| entry.project_path(cx))
```

Comment:
Both code places use the same `project_path` which is wrong.
What is meant to happen is an absolute path retrieval? You have to do `worktree.absolutize` (there's one place in the pane.rs that does this).

Reply:
Right I said that
> Right now, both actions do the same thing (i.e. copy relative path), as I need to find a way to get the absolute path
I now I know how to do that

Reply:
Now it copies absolute path!